source/API_Reference/Web_API_v3/cancel_schedule_send.apiblueprint

```diff
@@ -0,0 +1,152 @@
+---
+layout: page
+weight: 0
+title: Cancel Scheduled Sends
+navigation:
+  show: true
+---
+
+FORMAT: 1A
+
+# Cancellation of Scheduled Sends
+Cancelling scheduled send
+
+# Group Batch IDs
+
+If you set the [SMTPAPI header batch_id]({{root_url}}/API_Reference/SMTP_API/scheduling_parameters.html),
+it allows you to then cancel a scheduled send based on that batch_id by calling the [Cancel Scheduled Send endpoint]({{root_url}}/API_Reference/Web_API_v3/cancel_scheduled_send.html#Cancel-Scheduled-Sends)
+
+## Batch ID Generation [/mail/batch]
+Generate a new Batch ID to associate with scheduled sends
+
+### Generate Batch ID [POST]
+
++ Request
+
+
++ Response 201 (application/json)
+
+    + Body
+
+            {
+                "batch_id": "YOUR_BATCH_ID"
+            }
+
+## Validate Batch ID [/mail/batch/]
```

Comment:
just being nit-picky, we sometimes having a trailing slash, and sometimes we don't. I'm not sure if it makes any difference in the docs